detekt.yml

```diff
@@ -1,929 +1,64 @@
 build:
-  maxIssues: 0
```

Comment:
I could not find a default value for the maxIssues config. 
Removing this would mean that detekt would no longer cause any builds to fail on issues.
If this is intended, we could probably start weighting the issues below normally.

Reply:
As far as I was able to trace it, the default baseline configuration sets `maxIssues` to `0`. With the [example from the official documentation](https://detekt.dev/docs/introduction/configurations/#build-failure) it should fail if more than the number of wighted issues are found. I was also able to confirm in my tests by setting the weight for `UnusedParameter` to `1`.

On the second part of your comment, I refer to [my other post](https://github.com/Fraunhofer-AISEC/codyze/pull/828#issuecomment-1999047726).